### PR TITLE
updated changelog and revision for next development version

### PR DESCRIPTION
**Pull request by belagertem**
- updated revision to 8.1.0
- added `Unreleased` changelog section

# Checklist

The following aspects have been respected by the author of this pull request, confirmed by both pull request assignee **and** reviewer:

* Adherence to coding conventions
  * [x] Pull Request Assignee
  * [x] Reviewer
* Adherence to javadoc conventions
  * [x] Pull Request Assignee
  * [x] Reviewer
* Changelog update (necessity checked and entry added or not added respectively)
  * [x] Pull Request Assignee
  * [x] Reviewer
* README update (necessity checked and entry added or not added respectively)
  * [x] Pull Request Assignee
  * [x] Reviewer
* config update (necessity checked and entry added or not added respectively)
  * [x] Pull Request Assignee
  * [x] Reviewer
* SDCcc executable ran against a test device (if necessary)
  * [x] Pull Request Assignee
  * [x] Reviewer

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,8 @@ All notable changes to SDCcc will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## Unreleased
```
Comment (maximilianpilz) [inline CHANGELOG.md:7]: Please add an opening and closing bracket.

File: pom.xml [hunk 2]
```diff
@@ -15,7 +15,7 @@
     </modules>
 
     <properties>
-        <revision>8.0.1</revision>
+        <revision>8.1.0</revision>
```
Comment (maximilianpilz) [inline pom.xml:18]: I think it is rather unlikely that we will do such a release.

**Review by maximilianpilz — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (belagertem) [reply CHANGELOG.md]: done

File: pom.xml (hunk 2 above)
Comment (belagertem) [reply pom.xml:18]: What version would you suggest?

Comment (maximilianpilz) [reply pom.xml:18]: A major version increment, but we might also do that later, when we have more details on the next release.

**Review by maximilianpilz — APPROVED** [review]

**Pull request merged**